client-ts/Microsoft.AspNetCore.SignalR.Client.TS/Observable.ts

```diff
@@ -10,16 +10,39 @@ export interface Observer<T> {
     complete?: () => void;
 }
 
+export class ObserverDisposable<T> {
```

Comment:
Should this just be called Subscription?

Reply:
I think it makes more sense as Subscription. It's the result of calling subscribe. It's the Rx convention as well I believe